user-guide/Advanced_Functionality/DataMiner_Systems/Removing_a_DataMiner_Agent_from_a_DataMiner_System.md

```diff
@@ -19,7 +19,12 @@ Depending on whether the DataMiner Agent is part of a Failover pair or not, a di
 
 1. In the confirmation box, click *Yes*.
 
-1. If you want to keep using the removed DMA as a standalone Agent, restart the DMA.
+1. Stop the Agent you have removed and decommission it.
+
+1. In case your system is cloud connected, go to [Admin.Dataminer.Services](https://admin.dataminer.services/) and navigate further to the DxMs page. Here you see an overview of all the agents with an option to remove a node from the Cloud system. Click the remove button next to the agent you have just removed and confirm.
```

Comment:
We should avoid the term "cloud-connected", as this is ambiguous (the DataMiner cloud or some other cloud?). I'd also split this over several steps, as it's better to make sure that one step corresponds with just one action.
```suggestion
1. If your system is connected to dataminer.services, open the [Admin app](https://admin.dataminer.services/) and go to the DxMs page.

  This page shows an overview of all Agents or "nodes", with the option to remove a node from the system.

1. Click the garbage can icon next to the Agent you have just removed and confirm.
```

Reply:
ok with cloud connected, but for me personally this action is one step with several parts. (but no strong opinion, I'll leave it up to you)